src/latexify/codegen/function_codegen.py

```diff
@@ -141,42 +141,99 @@ def visit_If(self, node: ast.If) -> str:
 
     def visit_Match(self, node: ast.Match) -> str:
         """Visit a Match node"""
-        if not (
-            len(node.cases) >= 2
-            and isinstance(node.cases[-1].pattern, ast.MatchAs)
-            and node.cases[-1].pattern.name is None
-        ):
-            raise exceptions.LatexifySyntaxError(
-                "Match statement must contain the wildcard."
-            )
-
-        subject_latex = self._expression_codegen.visit(node.subject)
-        case_latexes: list[str] = []
-
-        for i, case in enumerate(node.cases):
-            if len(case.body) != 1 or not isinstance(case.body[0], ast.Return):
-                raise exceptions.LatexifyNotSupportedError(
-                    "Match cases must contain exactly 1 return statement."
-                )
-
-            if i < len(node.cases) - 1:
-                body_latex = self.visit(case.body[0])
-                cond_latex = self.visit(case.pattern)
-                case_latexes.append(
-                    body_latex + r", & \mathrm{if} \ " + subject_latex + cond_latex
+        latex = r"\left\{ \begin{array}{ll} "
+        subject_latex = self.visit(node.subject)
+        for i, match_case in enumerate(node.cases):
+            if len(match_case.body) != 1:
+                raise exceptions.LatexifySyntaxError(
+                    "Multiple statements are not supported in Match nodes."
```

Comment:
```suggestion
                    "Multiple statements are not supported in a Match case."
```

Related to [this comment](https://github.com/google/latexify_py/pull/152#discussion_r1045041035) I got, but talking about match case is probably more intuitive.